client/app/cloud/project/compute/loadbalancer/cloud-project-compute-loadbalancer.controller.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+angular.module("managerApp")
+  .controller("CloudProjectComputeLoadbalancerCtrl", function ($q, $scope, $translate, CloudProjectComputeLoadbalancerService, OvhApiCloudProjectIplb, OvhApiIpLoadBalancing, ControllerHelper, CloudMessage, $stateParams, OvhApiMe, URLS) {
+
+    var self = this,
+        serviceName = $stateParams.projectId;
+
+    //Datas
+    self.table = {
+        loadbalancer : [],
+    };
+
+    // Order link
+    self.urls = URLS;
+    self.locale = "";
+
+    self.order = {
+        by      : 'serviceName',
```

Comment:
we don't pad properties value like that.

by: 'serviceName'
reserve: false

You should configure eslint correctly in your text editor.

Reply:
done